cmd/manager/main.go

```diff
@@ -50,6 +57,12 @@ const (
 	podSchedulerAnnotationKey = "storageos.com/scheduler"
 )
 
+var (
+	metricsHost               = "0.0.0.0"
+	metricsPort         int32 = 8383
+	operatorMetricsPort int32 = 8686
```

Comment:
These are the default ports for operator-sdk generated projects. We can change it to something more suitable for us.